shell.nix

```diff
@@ -0,0 +1,10 @@
+(import
```

Comment:
FYI we opted to remove `flake-compat` support in `fh` to trim the dependency tree. So we don't need `shell.nix` anymore.

Reply:
This is mostly because I want some amount of editor support and this is needed for Nix integration in VS Code to work properly for me. Hopefully this isn't an issue, it only introduces flake-compat as dep, at time of use, for someone that invokes the `shell.nix`.

Reply:
How does this work? You don't have any `flake-compat` in the flake.lock...

Reply:
Oh, duh. I must not have re-activated the shell env after re-basing.

If y'all aren't opposed, I'll burn the FOD fetch into shell.nix so it doesn't need to be in the flake's dep tree. But still enables me until I can make VSCode work with flake shells.

Reply:
Okay, just pushed this.